### DISCUSSION TITLE
Cara Menampilkan Total Postingan Bagaimana ya Kang?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#224] [Q&A] [answered]
kang bagaimana cara menampilkan total postingan keseluruhan pada sebuah halaman `pages` yang mana pada `state.php` ada di isi nilai chunk nya. saat ane panggil menggunakan `$pages->count` jumlah yang muncul dibatasi dengan nilai chunk nya

**Comment by taufik-nurrohman** [comment, marked as answer]
Coba:

~~~ php
echo $pages->parent->count();
~~~

Atau:

~~~ php
// 1 directory level
echo q(g(dirname($page->path), 'page');

// Recursive
echo q(g(dirname($page->path), 'page', true);
~~~
